pkg/core/blockchain.go

```diff
@@ -2758,6 +2758,11 @@ func (bc *Blockchain) GetCommittee() (keys.PublicKeys, error) {
 	return pubs, nil
 }
 
+// GetCommitteeAddress returns the script hash of the committee multi-sig contract.
+func (bc *Blockchain) GetCommitteeAddress() (util.Uint160, error) {
```

Comment:
We don't need this new blockchain API. It should only be exposed from native Neo.